### PR TITLE
CLOUDSTACK-9168: TestPath to check if wrong value is inserted into nics table netmask field when creating a VM.

### PR DESCRIPTION
**Pull request by meenakshim1**
CLOUDSTACK-9168: Updates automation testscript 'testpath_netmask.py'.

Reference to #1245

### CODE REVIEW

**Comment by meenakshim1** [comment]
**Test Result -**

Check Netmask value in database ... === TestName: test_01_netmask_value_check | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 140.163s

OK

File: test/integration/testpaths/testpath_netmask.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+            # Create an account
+            cls.account = Account.create(
+                cls.apiclient,
+                cls.testdata["account"],
+                domainid=cls.domain.id
+            )
+
+            # Create user api client of the account
+            cls.userapiclient = testClient.getUserApiClient(
```
Comment (pavanb018) [inline test/integration/testpaths/testpath_netmask.py]: Do you need this userapiclient ? I don't see you using it anywhere in the file ?

File: test/integration/testpaths/testpath_netmask.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+            cleanup_resources(cls.apiclient, cls._cleanup)
+        except Exception as e:
+            raise Exception("Warning: Exception during cleanup : %s" % e)
+
+    def setUp(self):
+        if self.skiptest:
+            self.skipTest(
+                "Test not to be run on %s" %
+                self.hypervisor)
+        self.apiclient = self.testClient.getApiClient()
```
Comment (pavanb018) [inline test/integration/testpaths/testpath_netmask.py]: Isn't this redundant (self.apiclient ) ? You should already have acess to apiclient as it is created in setupClass I guess ?

File: test/integration/testpaths/testpath_netmask.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+            0,
+            "Check if netmask attribute in nics table \
+            stores correct value")
+
+        # Step 2
+        netmask_id = qryresult_netmask[0][2]
+        netlist = list_networks(self.apiclient,
+                                account=self.account.name,
+                                domainid=self.account.domainid)
+
```
Comment (pavanb018) [inline test/integration/testpaths/testpath_netmask.py]: Can you please add an assertion to validate "netlist" is not empty?

File: test/integration/testpaths/testpath_netmask.py (hunk 1 above)
Comment (meenakshim1) [inline test/integration/testpaths/testpath_netmask.py]: Updated

File: test/integration/testpaths/testpath_netmask.py (hunk 2 above)
Comment (meenakshim1) [inline test/integration/testpaths/testpath_netmask.py]: Updated

File: test/integration/testpaths/testpath_netmask.py (hunk 3 above)
Comment (meenakshim1) [inline test/integration/testpaths/testpath_netmask.py]: Updated

**Comment by pavanb018** [comment]
LGTM based on the code review

**Comment by rajap9711** [comment]
LGTM

**Pull request merged**